lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -105,7 +105,7 @@ class HelpPanel extends React.Component {
                 ${p => p.theme.addTerriaPrimaryBtnStyles(p)}
               `}
             >
-              {"Take the tour"}
+              {t("helpPanel.takeTour")}
```

Comment:
Also here, is there a reason why we chose to not do a `i18n.exists` check? I might be missing something but just wanted to clarify this to make sure I understand everything. Is the translation always guaranteed to be there?

Reply:
this was just a string that wasn't i18ified, `i18next.exists` is needed only for configurable translation (loading from config files) terms so we avoid two issues mentioned in previous comment.